### DISCUSSION TITLE
Getting rid of symbols gutter only

### DISCUSSION THREAD

**Discussion by riza-khan** [LazyVim/LazyVim#4246] [Q&A]
This is my `options.lua` file:

```
vim.opt.number = false
vim.opt.relativenumber = false
vim.diagnostic.enable(false)
vim.diagnostic.config({ virtual_text = false, signs = false })

```
I'm looking for a minimalistic view and can't seem to get rid of the diagnostics gutter (though I've disabled the symbols) but at the same time not disable the git signs. Anyone have thoughts?

<img width="265" alt="2024-08-02_10-10-03" src="https://github.com/user-attachments/assets/42df4aaa-b695-4c6a-a571-2c0568889768">

**Comment by nounder** [comment]
Same here. Gutter takes 4 columns even though it's empty.
